sensor_msgs/PointCloud2 cloud
geometry_msgs/Point center
geometry_msgs/Point[] voxel_points
geometry_msgs/Vector3[] voxel_normals
geometry_msgs/Vector3 normal
geometry_msgs/Vector3[] vecs_axis
float32[] sizes
sensor_msgs/RegionOfInterest region
